data/dashboards/usage.yaml

```diff
@@ -49,7 +49,7 @@ items:
             count: 3065
             source: https://cdn.humanatlas.io/hra-dashboard-data/usage/nih3d-downloads.HRA_1.csv
   - type: GridContainer
```

Comment:
You can remove the grid container since it has only 1 column. Just move the vega containers to the same level as the image container